packages/dd-trace/test/appsec/iast/analyzers/hardcoded-secret-analyzer.spec.js

```diff
@@ -29,17 +29,22 @@ describe('Hardcoded Secret Analyzer', () => {
       testCase.samples.forEach((sample, sampleIndex) => {
         // sample values are arrays containing the parts of the original token
         it(`should match rule ${testCase.id} with #${sampleIndex + 1} value ${sample[0]}...`, () => {
+          const value = sample.join('')
+          const ident = testCase.type === 'NameAndValue' ? value.split(' = ')[0] : undefined
```

Comment:
You created a const for that:

```suggestion
          const ident = testCase.type === NameAndValue ? value.split(' = ')[0] : undefined
```

Reply:
forgotten :S